apps/site/assets/package.json

```diff
@@ -23,11 +23,11 @@
     "phoenix_html": "file:../../../deps/phoenix_html",
     "places.js": "^1.6.0",
     "react": "^16.8.1",
-    "react-async": "^10.0.0",
     "react-dom": "^16.8.1",
     "react-leaflet": "^2.2.1",
     "react-leaflet-fullscreen": "^1.0.1",
     "sifter": "^0.5.1",
+    "swr": "^0.1.18",
     "tether": "^1.3.4",
     "turbolinks": "git+https://github.com/paulswartz/turbolinks.git#82b188e",
     "use-interval": "^1.2.1",
```

Comment:
`use-interval` is not removed here because we have other things that use it (although eventually those other things should be converted to use SWR too).